libs/ledgerjs/packages/context-module/src/external-plugin/data/DappResponse.ts

```diff
@@ -0,0 +1,61 @@
+export interface DappResponse {
+  b2c: B2c;
+  abis: Abis;
+  b2c_signatures: B2cSignatures;
+}
+
+export interface B2c {
+  blockchainName: string;
+  chainId: number;
+  contracts: Contract[];
+  name: string;
+}
+
+interface Contract {
+  address: string;
+  contractName: string;
+  selectors: { [selector: string]: ContractSelector };
+}
+
+interface ContractSelector {
+  erc20OfInterest: string[];
+  method: string;
+  plugin: string;
+}
+
+interface Abis {
+  [address: string]: AbiFunction[];
+}
+
+export interface AbiFunction {
+  type: string;
+  stateMutability?: string;
+  name?: string;
+  inputs?: AbiInput[];
+  outputs?: AbiOutput[];
+  anonymous?: false;
+}
+
+interface AbiInput {
+  name: string;
+  internalType: string;
+  indexed?: boolean;
+}
+
+interface AbiOutput {
+  name: string;
+  internalType: string;
+  components?: unknown; // FIXME: type
```

Comment:
You can find backend side object model [here](https://github.com/LedgerHQ/crypto-assets-service/blob/master/modules/api/src/main/scala/co/ledger/cal/api/assets/v1/model/asset/ABI.scala#L117) if that helps